api/src/main/java/io/cloudevents/format/BinaryMarshaller.java

```diff
@@ -53,7 +53,7 @@ private BinaryMarshaller() {}
 		return new Builder<A, T, P, H>();
 	}
 	
-	public static interface AttributeMarshalStep<A extends Attributes, T, P, H> {
+	public interface AttributeMarshalStep<A extends Attributes, T, P, H> {
```

Comment:
Is it better no static?

Reply:
Nested interfaces are static by default so this modifier was redundant